Mathlib/Topology/ContinuousMap/Units.lean

```diff
@@ -45,13 +45,11 @@ def unitsLift : C(X, Mˣ) ≃ C(X, M)ˣ where
   left_inv f := by ext; rfl
   right_inv f := by ext; rfl
 
--- Porting note: add manually because `simps` used `inv` and `simpNF` complained
```

Comment:
This porting note looks like it matches #18942 exactly, perhaps we should add a link to that to the comment earlier (on line 28). And then it's not really a porting note any more!